apollo-router/src/axum_http_server_factory.rs

```diff
@@ -482,6 +483,13 @@ async fn health_check() -> impl IntoResponse {
     Json(json!({ "status": "pass" }))
 }
 
+// Process the headers to make sure that `VARY` is set correctly
+fn process_vary_header(headers: &mut HeaderMap<HeaderValue>) {
+    if headers.get(VARY).cloned().is_none() {
```

Comment:
```suggestion
    if headers.get(VARY).as_ref().is_none() {
```

(havent tested)

Reply:
I've gone with:
```
if headers.get(VARY).is_none() {
```
I had the cloned() originally because I was doing something that upset the borrow checker. I don't need it at all now. Good spot!